examples/test.html

```diff
@@ -58,13 +58,18 @@
   attribution: '&copy; <a href="http://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors, &copy; <a href="http://cartodb.com/attributions">CartoDB</a>'
   }).addTo(map);
 
+  var layers = [{
+      sql: 'select * from ne_10m_populated_places_simple_3',
```

Comment:
Oh, sorry, didn't realise you were adding passing it when asking for the mapconfig. All good then.